### ISSUE TITLE
a11y: Color Contrast - Pod Security Admissions

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13739]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1 
Main Content
- Cancel and Edit as YAML buttons (on default state) 
- Name and Description input fields (on hover and focus)

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- privileged, baseline, restricted (dropdown option in selected state when keyboard focus is on option and on hover) 

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Cancel, Edit as YAML and Create button (on focus)

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF
Ratio: 1.78:1
Main Content
- placeholder text such as A unique name, Any text you want that better describes this resource, Version (default: latest, Enter a comma separated list of usernames and so on"

"Foreground color: #BEC1D2
Background color: #FFFFFF
Contrast ratio: 1.78:1
Main Content
- Version (default: latest) input field
- Enter a comma separated list of usernames input field
- Enter a comma separated list of runtimeClasses input field
- Enter a comma separated list of namespaces input field "

"Pod Security Admission: Create
- ""Pod Security Admission:"" link

Repository: Create
- ""Repository:"" link

Git Repo - Add Repository 
- ""Git Repo:"" link 

Users Retention Settings
- ""Users:"" link 

User: Create
- ""User:"" link 
"

## Expected result

A persistent underline should be present for the links to distinguish it within the block of text.

The contrast ratio between the foreground border color of the form field and background color should be sufficient.

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 178 fails VPAT. No notes. Original comment:

> Color alone #3D98D3 is used to identify mentioned link. Even though sufficient color contrast is provided, the links do not have a persistent underline. This will make it difficult for low vision users and those with colorblindness to easily distinguish links within the block of text.

Issue 181 fails VPAT. No notes. Original comment:

> Some form control borders have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 182 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
>
> Instances:
> Foreground Text Color: #3B71E1
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> - edit as YAML (on focus and hover)
> - Cancel button (on focus and hover)
> 
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF
> Ratio: 1.78:1
> Main Content
> - placeholder text such as A unique name, Any text you want that better describes this resource, Version (default: latest, Enter a comma separated list of usernames and so on
> 
> Foreground Text Color: #FFFFFF
> Background Text Color: #3D98D3
> Ratio: 3.16:1 
> Main Content
> - privileged, baseline, restricted (dropdown option in selected state when keyboard focus is not on option) 
> 
>
>Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.
"
